### PR TITLE
Add KnowBe4

### PR DESCRIPTION
**Pull request by arnydo**
Info and png (optimized) added.

### CODE REVIEW

File: _data/education.yml [hunk 1]
```diff
@@ -220,6 +220,13 @@ websites:
       twitter: ISACANews
       facebook: ISACAHQ
 
+    - name: KnowBe4 
+      url: https://training.knowbe4.com/
```
Comment (mxxcon) [inline _data/education.yml]: How come you used training subdomain and not the main one or www?

Comment (arnydo) [reply _data/education.yml]: Once you create an account with KnowBe4 you are redirected to https://training.knowbe4.com and is where the MFA is configured. I thought that was most appropriate as the url considering.

Comment (Carlgo11) [reply _data/education.yml]: @arnydo no, we list the main url here. Please change the url to `https://www.knowbe4.com/`

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
How about this logo instead?
It's transparent and taken from their website. (also slightly smaller in size)

**Review by mxxcon — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**